ledger-tool/src/main.rs

```diff
@@ -1119,6 +1127,9 @@ fn main() {
                     bank.squash();
                     bank.clean_accounts();
                     bank.update_accounts_hash();
+                    if refreeze {
+                        bank.unsafe_freeze(true);
```

Comment:
how about guarding/warning with `bank.operating_mode()`? Is this ever intended to be run for mainnet-beta? This option looks scary because we run bunch of code which isn't be run twice: https://github.com/solana-labs/solana/pull/10765/files#diff-a7549f152920d85fb44e6a784b8e5e1fR813-R817

Reply:
Well, it would be nice to have something that is safe for all case. We can have another method to rewrite just the hash with the update hard-forks values without touching the accounts values at all.

Reply:
Ok. I made it safer I think.